src/plugins/pluginManager/proposal.ts

```diff
@@ -88,6 +90,9 @@ export class PluginManagerProposal extends Proposal<IPluginManagerProposalState>
       pluginRegistered: item.schemeFactory.schemeRegistered,
       pluginRemoved: item.schemeFactory.schemeRemoved,
       pluginToRegisterData: item.schemeFactory.schemeToRegisterData,
+      pluginToRegisterDecodedData: item.schemeFactory.schemeToRemove === NULL_ADDRESS ?
```

Comment:
schemeToRemove  can be not NULL for the case of plugin `replace` . 
this condition should be : `item.schemeFactory.schemeToRegisterName !== NULL_ADDRESS `

Reply:
When `schemeToRegisterName` equals `NULL_ADDRESS`?
Same checking in Alchemy and it's working as expected.

Reply:
the encoded/decode data depend solely on the schemeNameToRegister . see https://github.com/daostack/arc/blob/arc-factory/contracts/schemes/SchemeFactory.sol#L83